### PR TITLE
feat (DOCS-CODE-021): [authN] [sign-in] canonical SPA identity sample for user sign-in using in .NET Blazor WebAssembly

### PR DESCRIPTION
**Pull request by ferantivero**
closes: #08220, #1657312

## WHY

we wanted to create our canonical sample for user sign-in using a .NET Blazor WebAssembly. This sample is meant to be used from Microsoft Identity platform tutorials.

## WHAT Changed?

- add canonical .NET Blazor WebAssembly standalone app
- add user sign-in/out using `Microsoft.Authentication.WebAssembly.Msal` 
- add readme following recent guidelines

## HOW to test?

Follow the instructions in the `README.md` file.

### CODE REVIEW

File: src/sign-in-blazorwasm/wwwroot/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <title>blazorwasm</title>
+    <base href="/" />
+    <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
+    <link href="css/app.css" rel="stylesheet" />
+    <link href="blazorwasm.styles.css" rel="stylesheet" />
```
Comment (ckittel) [inline src/sign-in-blazorwasm/wwwroot/index.html:11]: I don't know much about how blazor WASM works, but curious where the `blazorwasm.styles.css` comes from (I don't see on the filesystem) and how that's related to `css/app.css` (which I see on file system)

**Review by ckittel — APPROVED** [review]
I mean, looks like a "hello world" Blazor WASM app, with MSAL auth added.  #MissionAccomplished, I imagine.  Only one small question (more likely a knowledge gap on my side).

I did NOT validate that the README or anything is "up to snuff" -- I'm going to let CD take care of that validation.

File: src/sign-in-blazorwasm/wwwroot/index.html (hunk 1 above)
Comment (ferantivero) [reply src/sign-in-blazorwasm/wwwroot/index.html:11]: :eyes:

image

Comment (ckittel) [reply src/sign-in-blazorwasm/wwwroot/index.html]: Good to know!

**Pull request merged**